src/core/include/openvino/core/type/float8_e4m3.hpp

```diff
@@ -155,3 +155,67 @@ float8_e4m3 float8_e4m3::operator/=(const T& other) {
 #    pragma warning(pop)
 #endif
 }  // namespace ov
+
+namespace std {
+template <>
+class numeric_limits<ov::float8_e4m3> {
+public:
+    static constexpr bool is_specialized = true;
+    static constexpr ov::float8_e4m3 min() noexcept {
+        return ov::float8_e4m3::from_bits(0b00001000);  // minimum positive normalized value
+    }
+    static constexpr ov::float8_e4m3 max() noexcept {
+        return ov::float8_e4m3::from_bits(0b01111110);
+    }
+    static constexpr ov::float8_e4m3 lowest() noexcept {
+        return ov::float8_e4m3::from_bits(0b11111110);
+    }
+    static constexpr int digits = 4;
+    static constexpr int digits10 = 0;
+
+    static constexpr bool is_signed = true;
+    static constexpr bool is_integer = false;
+    static constexpr bool is_exact = false;
+
+    static constexpr int radix = 2;
```

Comment:
`radix` and `digits` are related and these would be defined "closer".

Reply:
The order of the members follows the standard (https://cplusplus.com/reference/limits/numeric_limits/), 
it's also aligned with the order in float16 and bfloat16 files.